docs/api/source/overview.rst

```diff
@@ -1,18 +1,14 @@
 .. _edX Data Analytics API Overview:
 
-################################################
+################################
 edX Data Analytics API Overview
-################################################
+################################
 
 The edX Data Analytics API provides the tools for building applications to view
 and analyze student activity in your course.
 
-The edX Data Analytics API uses Representational State Transfer (REST) design
-principles and supports JavaScript Object Notation (JSON) data-interchange
-format. Our REST API is simple, lightweight and optimized.
-
-You can use the edX Data Analytics API for web, desktop, and mobile
-applications.
+The edX Platform APIs use REST design principles and support the JSON data-
+interchange format.
```

Comment:
if a line break occurs after a hyphen, there will be white space between the hyphen and the following word in the HTML output. To avoid this, you can manually put "data-" onto line 11.